var/spack/repos/builtin/packages/py-ipywidgets/package.py

```diff
@@ -21,8 +22,8 @@ class PyIpywidgets(PythonPackage):
     depends_on('python@2.7:2.8,3.3:', type=('build', 'run'))
     # pip silently replaces distutils with setuptools
     depends_on('py-setuptools', type='build')
-    depends_on('py-ipython@4:', type=('build', 'run'))
-    depends_on('py-ipython@4:5', type=('build', 'run'), when='^python@:3.2')
+    depends_on('py-ipython@4:', type=('build', 'run'), when='^python@3.3:')
+    depends_on('py-ipython@4:5.8.0', type=('build', 'run'), when='^python@:3.3')
```

Comment:
Actually I think the previous ipython deps were correct, I'm not sure why I thought they were wrong. Sorry about that, can you revert that change?

Reply:
Sure, no problem! Thanks for the review